docs/guides/jobs/tensorboard.md

```diff
@@ -0,0 +1,125 @@
+dashboards that are present in the Tensorboard UI.
+
+## Enabling Tensorboard in your job
+
+To enable Tensorboard in your job, you need to specify a `--tensorboard` flag 
+when you run the job. Tensorboard can be enabled for both CLI jobs and when running 
+Jupyter notebooks.
+
+### Example
+
+This code snipped will train an mnist model and also store the training summary 
+to a log directory.
+
+```
+git clone https://github.com/floydhub/tensorflow-examples
+cd tensorflow-examples/tensorboard
+
+# Initialize the current directory to an existing or new project
+floyd init mnist-tensorboard
+floyd run --tensorboard "python mnist_tensorboard.py --log_dir /output/mnist --max_steps 5000"
+```
+
+!!! important ""
+    Notice that the the `log_dir` parameter is set to a path in the `/output` directory.
+    On Floydhub, Tensorboard only watches the `/output` directory. So you need to send 
+    any data meant for Tensorboard to any directory under `/output` path.
+
+Now you can go to the dashboard view the project page.
+
+![Project Dashboard](../../img/tensorboard_dashboard.png)
+
+Click on the job that was just started. You will notice that the job page now has a link 
+to Tensorboard. Click on it to open the Tensorboard dashboard in a new tab.
+
+![Tensonboard URL](../../img/tensorboard_url.png)
+
+### Tensorboard Dashboard
+
+![Tensorboard Dashboard](../../img/tensorboard_main.png)
+
+You can see that the "SCALARS" tab of Tensorboard is logging the accuracy of the 
+training and test data along with some other values. You may need to click on the title 
+bars (like `accuracy_1`) for the graph to open.
+
+The reason why these values are appearing on the dashboard is because the 
+`mnist_tensorboard.py` code has the following lines:
+
+```
+tf.summary.scalar('cross_entropy', cross_entropy)
+...
+tf.summary.scalar('accuracy', accuracy)
+```
+
+You can read more about how to use Tensorboard to log additional information in 
+the [Tensorboard README](https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/tensorboard/README.md#key-concepts).
+
+Explore the other tabs in the Tensorboad dashboard like "IMAGES" and "GRAPHS".
+
+### Tensorboard Tabs
+
+![Tensorboard Images](../../img/tensorboard_image.png)
+
+The IMAGES dashboard shows the transformations happening to the mnist images
+in real time while the training is happening.
+
+![Tensorboard Graphs](../../img/tensorboard_graph.png)
+
+The GRAPHS dashboard shows a graphical representation of the layers in the model.
+You can click into each part of the model to get more details.
+
+!!! Note ""
+    Tensorboard feature is only available for Tensorflow environments. 
+    See [this](../environments.md) page for full list of Tensorflow environments you 
+    can use.
+
+### Stopping Tensorboard
+
+Tensorboard runs in the same instance where your code is running. It will be up only until 
+your job finishes. Tensorboard will also stop when the job stops in `Failed`, `Timeout` or
```

Comment:
I was trying to make these cases other than Success but I think it is better to make this list more explicit.